.gitignore

```diff
@@ -110,6 +110,7 @@ venv.bak/
 *.i64
 !rules/lib
 
-# hooks output
-style-checker-output.log
+# hooks/ci.sh output
+isort-output
```

Comment:
this should be `isort-output.log`

Reply:
🙈 As I have mentioned in the commit, this doesn't actually matter because the extension `.log` is already in this file. Anyway, I'll change it! 😉